resourcemgr/resourcemgr.c

```diff
@@ -1309,7 +1309,8 @@ TSS2_RC ResourceMgrSendTpmCommand(
                 goto SendCommand;
             }
 
-            if( foundEntryPtr->connectionId != cmdConnectionId )
+            if( foundEntryPtr->status.persistent == 0 &&
+                foundEntryPtr->connectionId != cmdConnectionId )
```

Comment:
If the persistent objects can't be shared between different connections to RM, then users will be very restricted to get the benefit of persistency of object since all access to the persistent objects have to be finished in a single connection. How to handle it if people want to create a persistent object in a connection and use it for later connections? Using persistent objet instead of savecontext/loadcontext for a object can avoid saving context files between connections and also make the execution faster.